src/f4enix/input/d1suned.py

```
@@ -365,19 +476,188 @@ def remove_irradiation_time(self, index: int) -> None:
+        ref_scenario_num: int,
+        cooling_time: float = 0.0,
+    ) -> pd.DataFrame:
+
+        df_irr_ref = self.irradiation_file_df()
+
+        # --- Check that scenario numbers exist ---
+        if not (1 <= ref_scenario_num <= self.nsc):
+            raise IndexError(
+                f"ref_scenario_num {ref_scenario_num} is out of range. "
+                f"Must be between 1 and {self.nsc}."
+            )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)